exp/runtime/hooks/api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"sigs.k8s.io/cluster-api/internal/runtime/catalog"
+)
+
+var (
+	// GroupVersion is group version identifying rpc services defined in this package
```

Comment:
```suggestion
	// GroupVersion is group version identifying Runtime Hooks defined in this package
```
for consistency

(same in the other two occurrences below, I assume rpc services is leftover from a previous iteration)